uportal-war/src/main/java/org/apereo/portal/security/provider/cas/CasAssertionSecurityContext.java

```diff
@@ -167,6 +200,23 @@ protected void copyAssertionAttributesToUserAttributes(Assertion assertion) {
             log.debug("Adding attribute '{}' from Assertion with value '{}'; runtime type of value is {}",
                     attrEntry.getKey(), attrEntry.getValue(), attrEntry.getValue().getClass().getName());
 
+            // Check for credential
+            if (decryptCredentialToPassword && cipher != null && attrEntry.getKey().equals(CREDENTIAL_KEY)) {
+                try {
+                    final String encPwd = (String) (attrEntry.getValue() instanceof List
+                            ? ((List) attrEntry.getValue()).get(0) : attrEntry.getValue());
+                    byte[] cred64 = Base64.getDecoder().decode(encPwd);
```

Comment:
🚧 Travis CI is throwing a compilation error for `cannot find symbol Base64` on line 208.

ref: https://travis-ci.org/Jasig/uPortal/builds/188714333#L2698-L2703